DATA/production/qc-async/ft0.json

```diff
@@ -3,43 +3,40 @@
       "config": {
         "database": {
           "implementation": "CCDB",
-          "host": "qcdb.cern.ch:8083",
+          "host": "ali-qcdb.cern.ch:8083",
           "username": "not_applicable",
           "password": "not_applicable",
           "name": "not_applicable"
         },
         "Activity": {
           "number": "42",
-          "type": "2"
-        },
+            "type": "2"
+       },
         "monitoring": {
-          "url": "influxdb-unix:///tmp/telegraf.sock"
+          "url": "influxdb-unix:///tmp/telegraf.sock" 
         },
         "consul": {
-          "url": "alio2-cr1-hv-aliecs.cern.ch:8500"
+          "url": "ali-qcdb.cern.ch:8083"  
         },
         "conditionDB": {
-          "url": "qcdb.cern.ch:8083"
+          "url": "ali-qcdb.cern.ch:8083"
         }
       },
       "tasks": {
-        "DigitQcTaskFT0": {
+        "RecPointsQcTask": {
           "active": "true",
-          "className": "o2::quality_control_modules::ft0::DigitQcTask",
+          "className": "o2::quality_control_modules::ft0::RecPointsQcTask",
           "moduleName": "QcFT0",
           "detectorName": "FT0",
-          "cycleDurationSeconds": "60",
+          "cycleDurationSeconds": "600",
           "maxNumberCycles": "-1",
-          "dataSource": {
-            "type": "direct",
-            "query": "digits:FT0/DIGITSBC/0;channels:FT0/DIGITSCH/0"
-          },
-          "taskParameters": {
-            "ChannelIDs": "0,1,2,3,4,5,6,7,8,9,10,11"
-          }
+            "dataSource": {
+              "type": "direct",
+              "query": "recpoints:FT0/RECPOINTS/0;channels:FT0/RECCHDATA/0"
+            },
+            "saveObjectsToFile":"qcobjects.root"
```

Comment:
I would propose to remove this, since it will create an object for each sub job on grid. This should be used only for debugging locally

Reply:
Then perhaps one could change the name to something more meaningful, so it is clear what creates this file?

Reply:
I changed name of file and removed consul url, as you wrote

Reply:
thank you!